data/tag_windows.txt

```diff
@@ -103,16 +329,294 @@ autorun
+  data_type is 'windows:evtx:record' AND source_name is 'MsiInstaller' AND (event_identifier is 11707 OR event_identifier is 11728 OR event_identifier is 1033 OR event_identifier is 1036)
+  data_type is 'windows:evtx:record' AND source_name is 'Microsoft-Windows-Application-Experience' AND event_identifier is 903
+  data_type is 'windows:evtx:record' AND source_name is 'Microsoft-Windows-Application-Experience' AND event_identifier is 904
+
+# Tags Windows application update events.
+application_update
+  data_type is 'windows:evtx:record' AND source_name is 'Microsoft-Windows-Application-Experience' AND event_identifier is 905
+  data_type is 'windows:evtx:record' AND source_name is 'Microsoft-Windows-Application-Experience' AND event_identifier is 905
+
+# Tags Windows application removal events.
+application_removal
+  data_type is 'windows:evtx:record' AND source_name is 'Microsoft-Windows-Application-Experience' AND (event_identifier is 907 OR event_identifier is 908)
+
+kb_install
+  data_type is 'windows:evtx:record' AND source_name is 'MsiInstaller' AND event_identifier is 1022
+
+# Process remote
+rpc_attempt
+  data_type is 'windows:evtx:record' AND source_name is 'Microsoft-Windows-Security-Auditing' AND event_identifier is 5712
+
+# DLL Search Order Hijacking  https://attack.mitre.org/techniques/T1038
+dll_search
+  data_type is 'windows:evtx:record' AND source_name is 'Microsoft-Windows-Sysmon' AND event_identifier is 7 AND xml_string contains '\\users\\' AND xml_string contains '.dll'
+  data_type is 'windows:registry:key_value' AND key_path contains 'CurrentControlSet\\Control\\Session Manager' AND values contains 'SafeDLLSearchMode'
+
+# sysmon named pipe
+named_pipe
+  data_type is 'windows:evtx:record' AND source_name is 'Microsoft-Windows-Sysmon' AND (event_identifier is 17 OR event_identifier is 18)
+
+ad_replica
+  data_type is 'windows:evtx:record' AND source_name is 'Microsoft-Windows-Security-Auditing' AND (event_identifier is 4928 OR event_identifier is 4929)
+
+# can indicate process use intensive (like miner)
+ram_consum
+  data_type is 'windows:evtx:record' AND source_name is 'Microsoft-Windows-Resource-Exhaustion-Detector' AND (event_identifier is 1001 OR event_identifier is 2004)
+
+# certiciate change https://attack.mitre.org/techniques/T1130
+cert_change
+  data_type is 'windows:registry:key_value' AND (key_path contains 'SOFTWARE\\Microsoft\\EnterpriseCertificates\\Root\\Certificates' OR key_path contains 'Microsoft\\SystemCertificates\\Root\\Certificates')
+  data_type is 'windows:evtx:record' AND source_name is 'Microsoft-Windows-Sysmon' AND (event_identifier is 13 OR event_identifier is 12) AND (xml_string contains 'SOFTWARE\\Microsoft\\EnterpriseCertificates\\Root\\Certificates' OR xml_string contains 'Microsoft\\SystemCertificates\\Root\\Certificates')
+
+registry_modified
+  data_type is 'windows:evtx:record' AND source_name is 'Microsoft-Windows-Security-Auditing' AND event_identifier is 4657
+
+ip_change
```

Comment:
Provide some context what you're labeling here

````
Microsoft-Windows-Iphlpsvc 4200 - Unable to read or write the NTE Context list for network adapter %2. IP interfaces on this network adapter may not be initialized completely.
```

Reply:
@lprat this does not address my question. Please add some explanation what type of events this tagging rule is labeling and how to interpret the corresponding event log entries as "ip_change"

Reply:
@lprat please correct this, this does not correspond with https://community.spiceworks.com/topic/775081-ms-server-event-id-4200-source-iphlpsvc-share-connectivity-help

Reply:
Sorry, I had not seen your questions in december. 
Tag "IP_change" is error! I had tagged with this name because I thought identified the event that indicated the IP address that the system takes when it connects to the network.